### DISCUSSION TITLE
Lost Agent Count

### DISCUSSION THREAD

**Discussion by wangshiok** [amidaware/tacticalrmm#1304] [General]
![image](https://user-images.githubusercontent.com/113150961/194266592-867748e5-bf4c-4da3-9ab9-ec094b279420.png)

**Comment by silversword411** [comment]
Tools menu | Clear cache
Still 0?

Are you a multi-user system? We need a how to replicate set of steps

**Comment by wh1te909** [comment]
agent count is handled by the django websocket service (daphne). what's the output of `sudo systemctl status --full --no-pager daphne.service`

**Comment by wangshiok** [comment]
> Tools menu | Clear cache Still 0? still 0
> 
> Are you a multi-user system? - single user

We need a how to replicate set of steps.  used a HyperV check-point to rollback to V14.6 from V14.7, after rollback, lost agent count, then upgraded to V15.0 from 14.6, still 0

**Comment by wangshiok** [comment]
> agent count is handled by the django websocket service (daphne). what's the output of `sudo systemctl status --full --no-pager daphne.service`

● daphne.service - django channels daemon
     Loaded: loaded (/etc/systemd/system/daphne.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) since Fri 2022-10-07 19:13:37 AEDT; 2s ago
    Process: 2871273 ExecStart=/rmm/api/env/bin/daphne -u /rmm/daphne.sock tacticalrmm.asgi:application (code=exited, status=0/SUCCESS)
   Main PID: 2871273 (code=exited, status=0/SUCCESS)
        CPU: 2.083s

Oct 07 19:13:41 tactical-server systemd[1]: daphne.service: Scheduled restart job, restart counter is at 137601.
Oct 07 19:13:41 tactical-server systemd[1]: Stopped django channels daemon.
Oct 07 19:13:41 tactical-server systemd[1]: daphne.service: Consumed 2.083s CPU time.
Oct 07 19:13:41 tactical-server systemd[1]: Started django channels daemon.
Oct 07 19:13:43 tactical-server daphne[2871468]: 2022-10-07 08:13:43,964 INFO     Starting server at unix:/rmm/daphne.sock
Oct 07 19:13:43 tactical-server daphne[2871468]: 2022-10-07 08:13:43,965 INFO     HTTP/2 support not enabled (install the http2 and tls Twisted extras)
Oct 07 19:13:43 tactical-server daphne[2871468]: 2022-10-07 08:13:43,965 INFO     Configuring endpoint unix:/rmm/daphne.sock
Oct 07 19:13:43 tactical-server daphne[2871468]: 2022-10-07 08:13:43,966 CRITICAL Listen failure: [Errno 1] Operation not permitted

**Reply by wh1te909** [reply]
see https://github.com/amidaware/tacticalrmm/issues/1210 and manually apply the fix until I update the configs
